### ISSUE TITLE
Screenreader accessibility for jump menu

### ISSUE THREAD

**Issue by benerachel** [benerachel/d4i#9]
As a user with visual impairment I want to navigate form content via the menu while using a screen reader. 

The navigation drawer is completely inaccessible with screen reader. Screen reader does not read the label of the navigation drawer button. After the drawer is expanded, the screen reader focus does not move to the first item of the drawer. Instead, the focus remains on the main page.

Screen reader should have:
• Announced "Navigation Drawer" when the focus is on the   button.
• Announce the state of the drawer, i.e., collapsed or expanded. 
• Keyboard accessible.
• Proper contrast.
